### PR TITLE
Restructured downloads.md

### PR DESCRIPTION
**Pull request by toby63**
I restructured the downloads.md a bit, including:
* added sections for windows, mac os & linux
* Linux: added a link to the mumble wiki for other distros
* Added a section for "Downloads & Sources on Github"
* Renamed "third party clients" to "mobile clients"
* added seperate sections for iOS and Android
* added notes about third-party apps

The intention is better readability and a better possibility for notes to be added to each section (which is harder when everything is in one block).

Also (partly) solves: #89 (for linux) #61 (ones the wiki is updated)

### CODE REVIEW

File: hugo/content/downloads.md [hunk 1]
```diff
@@ -16,18 +18,27 @@ Version 1.3.0 is the latest stable version of Mumble and was released on Septemb
             <span class="os-name">Windows 32-bit</span>
         </a>
     </div>
+</div>
+    
+### Mac OS ###
```
Comment (davidebeatrici) [inline hugo/content/downloads.md:23]:
```suggestion
### macOS ###
```

File: hugo/content/downloads.md [hunk 2]
```diff
@@ -39,39 +50,55 @@ Version 1.3.0 is the latest stable version of Mumble and was released on Septemb
...
-            <span class="os-name">Static macOS Server</span>
-        </a>
-    </div>
+    </div>    
 </div>
 
-For the individual files and for a zipped sources file see the [1.3.0 GitHub release page](https://github.com/mumble-voip/mumble/releases/tag/1.3.0).
+For other Linux Distributions and more information, see the [Installation Page in Mumble's Official Wiki](https://wiki.mumble.info/wiki/Installing_Mumble#Linux).
+
+## Downloads & Source Code on Github ##
```
Comment (davidebeatrici) [inline hugo/content/downloads.md:58]:
```suggestion
## Downloads & Source code on Github ##
```

File: hugo/content/downloads.md [hunk 3]
```diff
@@ -39,39 +50,55 @@ Version 1.3.0 is the latest stable version of Mumble and was released on Septemb
...
+<div class="download-box">
+        <a href="ios">
+            <span class="os os-ios" aria-hidden="true"></span>
+            <span class="os-name">iOS</span>
+        </a>
     </div>
-    <div class="download-box">
+</div>
+
+#### Third-Party App ####
```
Comment (davidebeatrici) [inline hugo/content/downloads.md:77]:
```suggestion
#### Third-Party app ####
```

File: hugo/content/downloads.md [hunk 4]
```diff
@@ -39,39 +50,55 @@ Version 1.3.0 is the latest stable version of Mumble and was released on Septemb
...
-        <a href="osx-static-server">
-            <span class="os os-mac" aria-hidden="true"></span>
-            <span class="os-name">Static macOS Server</span>
-        </a>
-    </div>
+    </div>    
 </div>
 
-For the individual files and for a zipped sources file see the [1.3.0 GitHub release page](https://github.com/mumble-voip/mumble/releases/tag/1.3.0).
+For other Linux Distributions and more information, see the [Installation Page in Mumble's Official Wiki](https://wiki.mumble.info/wiki/Installing_Mumble#Linux).
```
Comment (davidebeatrici) [inline hugo/content/downloads.md:56]:
```suggestion
For other Linux distributions and more information, see the [Installation page in Mumble's official wiki](https://wiki.mumble.info/wiki/Installing_Mumble#Linux).
```

File: hugo/content/downloads.md [hunk 5]
```diff
@@ -16,18 +18,27 @@ Version 1.3.0 is the latest stable version of Mumble and was released on Septemb
...
             <span class="os-name">macOS</span>
         </a>
     </div>
     <div class="download-box">
-        <a href="ios">
-            <span class="os os-ios" aria-hidden="true"></span>
-            <span class="os-name">iOS</span>
+        <a href="osx-static-server">
+            <span class="os os-mac" aria-hidden="true"></span>
+            <span class="os-name">Static macOS Server</span>
```
Comment (davidebeatrici) [inline hugo/content/downloads.md:34]:
```suggestion
            <span class="os-name">Static macOS server</span>
```

**Comment by Kissaki** [comment]
I want to review this before landing as well.

**Comment by Kissaki** [comment]
For the Git commits why do you use the pseudonymized GitHub author as the commit author but then use the `Signed-off-by` with specific information about you instead of using your non-pseudonym information as the author right away?

**Comment by toby63** [comment]
Sry, I simply used the github editor and sometimes I forget to add the author information :smile: 
I see you already rebased it.

**Comment by Kissaki** [comment]
Alright, so I take you will be fine with me cleaning it up, moving the signed off to the author field.

I see you added categories to the downloads. Looking at the downloads I actually also saw this as a good idea. But I think it can be a bit better than your suggestion. I will play around with it a bit more.

I also want to implement a suggest-users-os-download.

I think the distinction between official and third party clients is an important one. I don’t think they should be mixed together too much. That will lead to user confusion, and misunderstanding of what we support/what to report to us.

But splitting off the mobile clients as you suggest makes sense I think.

**Comment by toby63** [comment]
> Alright, so I take you will be fine with me cleaning it up, moving the signed off to the author field.

Yes.

> I also want to implement a suggest-users-os-download.

I don't like those, but maybe you could make sure, that it is still possible to choose other OS's.

> I think the distinction between official and third party clients is an important one. I don’t think they should be mixed together too much.

While I agree, there are only two for now and with the Note it should be enough.
If you want to clearly seperate them from official download, don't put them on the website, because people might still assume that it belongs to the project.

Also you might not suggest such very old releases to your users.
iOS seems to be very outdated.

**Comment by Kissaki** [comment]
Of course. The idea is to make the standard path easy to use and spot while allowing to decide differently and choose manually.

**Comment by toby63** [comment]
Could we merge this before you release 1.3.1?

Also consider two things:
1. delete Mumblefy
2. Add a note that the iOS version is older (and not release 1.3.1)

**Comment by Kissaki** [comment]
Thank you for your suggestions and inspiration.

I tried a few things but ultimately ended up in a categorisation half-way to yours. I did not add them where not necessary though in order not to unnecessarily bloat the page and its vertical space use.

You added a link to the install wiki page. But I decided to integrate the useful information from there on in the content instead.

I also added some more information about various things.

**Comment by Kissaki** [comment]
I credited you as a co-author in the commit.

**Comment by toby63** [comment]
Just some suggestions:
* put this
`A Linux distribution may provide their own packages and have their own maintainer, which we will describe below. We also link to some third party projects.`
and the linux boxes in the section for linux.
* The ubuntu ppa mention is doubled, as the ubuntu box also points to the ppa
* I would have still added a section for Github and Source Code

**Comment by Kissaki** [comment]
Yeah I left the double ubuntu and PPA for now. They are separate but they each do a slightly different thing. And they present differently; with an icon and command line. For now I will leave it like this until the next iteration of improvements. Because I don't want to move the icon down as-is, as the big icon it is.

Yeah I guess a Repository and source link would be good.

**Pull request closed without merging**